manifests/addons/dashboards/istio-extension-dashboard.json

```diff
@@ -82,7 +82,7 @@
         {
           "expr": "avg(envoy_wasm_vm_null_created)",
           "interval": "",
-          "legendFormat": "null",
+          "legendFormat": "native",
```

Comment:
I think native is a better name to show on the graph. Let me know if you think otherwise @mandarjog @kyessenov

Reply:
Agree

Reply:
@carolynhu can you move "active" before "created" ?

I think "native" is better terminology, but we use `nullvm` to describe the concept so perhaps that is still a better label.